.github/workflows/test-and-build.yml

```diff
@@ -0,0 +1,95 @@
+---
+name: Build and Push Up Feature or Fix Branch Image
+on:
```

Comment:
If you would, please add `workflow_dispatch` as a trigger as well. Granted, it's easy enough to make a branch named `fix-*` to build from, but having `workflow_dispatch` is really useful for updating actions.